.github/workflows/scripts/prs/checkTypeLabel.js

```diff
@@ -10,6 +10,7 @@ const createEnumerationFromArray = (stringArray) =>
     : stringArray.map((s) => `\`${s}\``).join('');
 
 const typeLabels = [
+  'docs',
```

Comment:
I'm not sure about this one. I believe I always add one of "bug" or "enhancement" when I do docs changes.

Reply:
I added it because if this discussion: https://mui-org.slack.com/archives/C011VC970AW/p1742244212315709